src/main/java/me/totalfreedom/totalfreedommod/command/Command_ro.java

```diff
@@ -50,7 +50,7 @@ else if (args[0].equalsIgnoreCase("banners") || args[0].equalsIgnoreCase("banner
 
                 if (fromMaterial == null || fromMaterial == Material.AIR || !fromMaterial.isBlock())
```

Comment:
You've missed line 20 on this where the defaults have been removed elsewhere.

Reply:
I am not sure what you mean by this, line 20 is just
@CommandPermissions(level = Rank.ADMIN, source = SourceType.BOTH)

Reply:
fixed